app/views/users/sessions/new.html.erb

```diff
@@ -1,7 +1,8 @@
+<%= h1 "Sign in", size: "xl" %>
```

Comment:
???

Reply:
This lines up with other forms that put the `h1` above the `form`, and the errors continue to render above the heading.